ctapipe/io/tableloader.py

```diff
@@ -162,16 +162,14 @@ def read_subarray_events(self, labels=None, sort_by=["obs_id", "event_id"]):
             Single table containing the enabled data information."""
         
         if not any(self.all_loads):
-            self.log.critical("No data enabled for loading.")
             self.close()
-            return None # it should probably return some exit status != 0
+            raise ValueError("No data enabled for loading.")
 
         if (sum(self.all_loads)>1) and self.load_instrument and (not any([self.load_dl1_images, self.load_dl1_parameters, self.load_true_images])):
```

Comment:
No need to check `sum > 1` since you did that in the previous `if not any` statement

Reply:
removed

Reply:
actually, it was right before, so I reverted it (in fact the unit test was failing again):
the >1 ensures that there is _at least_ `self.load_instrument`, since it will crash if joining it with any combination of event-based data (e.g. trigger+ simulation, dl2 alone, dl2+ trigger, etc...)